README.md

```diff
@@ -147,6 +147,17 @@ default `'text'`)
 * `VERBOSE` - Maximize log verbosity if non-empty (equivalent to a level
 of `MonoLogger::DEBUG`, default `false`)
 
+*Warning: If you use a different redis database, other than the default database (0), to store your redis jobs, you will have store the Resque.redis into the initializer file*
```

Comment:
This isn't necessarily true, though.  I think it would be clearer to say that specifying the full Redis URL within the schedule process is important and perhaps give some examples of how to do this, just like in [the resque web example](examples/config/initializers/resque-web.rb).  I would also support adding the Redis URL as an env/cli option so that initializers are not necessary.